lib/rust/ensogl/core/src/display/object/instance.rs

```diff
@@ -1814,6 +1750,11 @@ impl Model {
                 }
             } else {
                 trace!("Self origin and layers did not change.");
+
+                if self.dirty.computed_size.check() {
+                    self.on_updated_source.emit(());
```

Comment:
I don't understand the name `on_updated_source` - what does it mean? I see no docs next to this field. I know its not part of this PR, sorry for catching it late.

Reply:
Looks like "A Source (i.e. frp::Source) for `on_updated`" event.

Reply:
oh